bundler.d/fog.rb

```diff
@@ -1,5 +1,5 @@
 group :fog do
- gem 'fog', '1.25.0'
- gem 'fog-core', '1.25.0'
+ gem 'fog', '1.26.0'
```

Comment:
we normally have a separate issue for fog updates, this can probably be taken off during merge..
which version of fog is minimal for this feature?